02-peer-protocol.md

```diff
@@ -163,17 +163,17 @@ The `NodeStateUpdate` message is used to tell a peer about a change in the node
     `repeated string pairs = 2`
     The sender's list of trading pair symbols, constructed with the base currency first, followed by a  '/' separator and the quote currency (e.g., [“LTC/BTC”, “DAI/BTC”])
 
-    `string raiden_address = 3`
-    The sender's Raiden address
+    `string connext_address = 3`
+    The sender's connext address
 
     `map<string, string> lnd_pub_keys = 4`
     The sender's list of LND public keys
 
     `map<string, string> token_identifiers = 5`
-    Mapping between currency symbols to token identifiers such as LND chains or Raiden token contract address (e.g., { BTC: 'bitcoin-testnet', LTC: 'litecoin-testnet' })
+    Mapping between currency symbols to token identifiers such as LND chains or connext token contract address (e.g., { BTC: 'bitcoin-mainnet', LTC: 'litecoin-mainnet' })
```

Comment:
Perhaps ethereum ERC20 token contract address or zero address `0x0000000000000000000000000000000000000000` for ETH is better here?

Reply:
Like this? If not, please point me to the code snippet in xud where we do that mapping for ETH-ERC20.